src/main/java/at/hannibal2/skyhanni/features/event/chocolatefactory/ChocolateFactoryAPI.kt

```diff
@@ -316,13 +318,14 @@ object ChocolateFactoryAPI {
         prestigeIndex = data.prestigeIndex
         milestoneIndex = data.milestoneIndex
         leaderboardIndex = data.leaderboardIndex
+        handCookieIndex = data.handCookieIndex
         timeTowerIndex = data.timeTowerIndex
+        shrineIndex = data.shrineIndex
+        coachRabbitIndex = data.coachRabbitIndex
         maxRabbits = data.maxRabbits
-
-        val disabledFeatures = event.getConstant<DisabledFeaturesJson>("DisabledFeatures")
     }
 
-    private fun List<String>.getUpgradeCost(): Long? {
+    fun List<String>.getChocolateUpgradeCost(): Long? {
```

Comment:
This cant be public. otherwise this extenstion function is avaliable to every List<String> everywhere. rather make it not a extension function.